airbyte-integrations/connectors/source-sentry/source_sentry/streams.py

```diff
@@ -69,14 +71,21 @@ def __init__(self, *args, **kwargs):
         super(SentryIncremental, self).__init__(*args, **kwargs)
         self._cursor_value = None
 
+    def validate_state_value(self, state_value: str = None) -> str:
```

Comment:
Let's add unit test for this two methods below 😅

Reply:
Done